runner/database.go

```diff
@@ -67,6 +67,7 @@ var defaultPorts = map[DatabaseConnectorInfoType]string{
 	YugabyteDatabase:      "5433",
 	QuestDatabase:         "8812",
 	Neo4jDatabase:         "7687",
+	ODBCDatabase:          "1433",
```

Comment:
This isn't a reasonable port but nothing is so maybe it doesn't matter.

Reply:
It's actually mssql default port